frontend/app/helpers/spree/frontend_helper.rb

```diff
@@ -214,7 +214,14 @@ def price_filter_values
     def filtering_params
       static_filters = %w(keywords price sort_by)
 
-      @option_types.map(&:filter_param).concat(static_filters)
+      available_option_types.map(&:filter_param).concat(static_filters)
+    end
+
+    def available_option_types
+      @available_option_types ||= Rails.cache.fetch('available-option-types', expires_in: 3.minutes) do
+        Spree::OptionType.includes(:option_values).to_a
+      end
+      @available_option_types
```

Comment:
It's a minor issue, but this `@available_option_types` isn't necessary.

Reply:
@piotrleniec-spark it's for readability